internal/x/bytes/transform.go

```diff
@@ -0,0 +1,64 @@
+// Copyright (c) 2017-present SIGHUP s.r.l All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package bytesx
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"regexp"
+	"time"
+
+	logrusx "github.com/sighupio/furyctl/internal/x/logrus"
+)
+
+type TransformFunc func([]byte) ([]byte, error)
+
+// This constant was taken from the following public repository:
+// Name: stripansi
+// URL: https://github.com/acarl005/stripansi
+// Commit: 5a71ef0e047df0427e87a79f27009029921f1f9b
+// Author: https://github.com/acarl005
+// License: MIT License.
+
+const ansi = "[\u001B\u009B][[\\]()#;?]*(?:(?:(?:[a-zA-Z\\d]*(?:;[a-zA-Z\\d]*)*)?\u0007)|(?:(?:\\d{1,4}(?:;\\d{0,4})*)?[\\dA-PRZcf-ntqry=><~]))" //nolint:lll // Cannot split regex in multiple lines.
+
+var ErrJSONTransform = errors.New("error while transform to json")
+
+func StripColor(p []byte) ([]byte, error) {
+	reg := regexp.MustCompile(ansi)
+
+	s := string(p)
+
+	strippedS := reg.ReplaceAllString(s, "")
+
+	return []byte(strippedS), nil
+}
+
+func ToJSONLogFormat(level string, action *string) TransformFunc {
```

Comment:
why is action a pointer? does it need to be modified somewhere down the road?

Reply:
My idea was to prepare this function to be also used from logrus as a JSON format instead of the default (poorly customizable) one, so the action will be an optional parameter but you are right, maybe it would be better to have the possibility to pass an empty string and handle that instead of a pointer that can mutate.